### PR TITLE
[BH-1809][BH-1835] Add date format setting

### PR DESCRIPTION
**Pull request by Maciej-Mudita**
Added date format selection between YYYY/DD/MM and YYYY/MM/DD. The time setting has been updated.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: products/BellHybrid/apps/application-bell-settings/widgets/TimeSetListItem.hpp [hunk 1]
```diff
@@ -9,12 +9,12 @@
...
 namespace gui
 {
-    class TimeSetFmtSpinner;
+    class TimeSetSpinner;
 
     class TimeSetListItem : public BellSideListItem
     {
       public:
-        TimeSetFmtSpinner *timeSetFmtSpinner = nullptr;
+        TimeSetSpinner *timeSetSpinner = nullptr;
```
Comment (dawidwojtas-mudita) [inline products/BellHybrid/apps/application-bell-settings/widgets/TimeSetListItem.hpp:17]: Use `{}` as you did before in `DateSetSpinner.hpp`

**Review by dawidwojtas-mudita — APPROVED** [review]
Good job ;)

**Review by Lefucjusz — APPROVED** [review]

File: products/BellHybrid/apps/application-bell-settings/widgets/TimeSetListItem.hpp (hunk 1 above)
Comment (Maciej-Mudita) [reply products/BellHybrid/apps/application-bell-settings/widgets/TimeSetListItem.hpp:17]: done

File: module-apps/apps-common/widgets/DateSetSpinner.cpp [hunk 2]
```diff
@@ -26,221 +27,124 @@ namespace gui
...
+    std::uint32_t DateSetSpinner::getWidgetMinimumAreaWidth() const
     {
-        return day->widgetMinimumArea.w + firstSlash->widgetMinimumArea.w + month->widgetMinimumArea.w +
-               secondSlash->widgetMinimumArea.w + year->widgetMinimumArea.w;
+        return (type == Type::year) ? year->widgetMinimumArea.w : dayMonth->widgetMinimumArea.w;
+    }
+
+    std::uint16_t DateSetSpinner::getFontHeight(const std::string &fontName) const
+    {
+        const RawFont *font = FontManager::getInstance().getFont(fontName);
```
[the remark this answers is not in the document]
Comment (Maciej-Mudita) [reply module-apps/apps-common/widgets/DateSetSpinner.cpp]: done

**Pull request merged**